### DISCUSSION TITLE
Dns resolution failure inside pod with ubuntu 22.04 and two ethernet port CALICO CNI

### DISCUSSION THREAD

**Discussion by ambou75** [rancher/rke2#6424] [Q&A]
Hello,

I am trying to install rke2 with cni calico on linux nodes with two network cards, my operating system is ubuntu 22.04 LTS. I have an eth0 card for the kubernetes cluster network and an eth1 card for future NFS connections for my PVCs. the eth1 card is only configured on my worker nodes (the master nodes only having important roles and no workload).

When I attempt DNS resolution from a test pod, located on a worker node, I get no resolution response, even when trying to resolve kubernetes.default internally.

When I deactivate the second card, DNS resolution is operational.

I should point out that the second eth1 card does not have a default gateway, it is carried by the eth0 card which has the gateway and a DNS server, DNS resolutions from the host work very well.

The same configuration with the default cni canal and two cards poses no problem

Here is my rke2 installation configuration via the tarball method with the use of the calico plugin.

![Capture](https://github.com/user-attachments/assets/74de93e1-92c4-45ed-84f1-c4c1613228d7)

Am I using the correct method to install rke2 with the calico plugin?

Thank you in advance for your feedback

**Comment by cengizhanR** [comment]
Hello, Out of topic, How did you manage this " NFS connections for my PVC"?
